apps/api/v1/pages/api/bookings/[id]/_get.ts

```diff
@@ -89,9 +90,21 @@ import { schemaQueryIdParseInt } from "~/lib/validations/shared/queryIdTransform
 export async function getHandler(req: NextApiRequest) {
   const { query } = req;
   const { id } = schemaQueryIdParseInt.parse(query);
+
+  const queryFilterForExpand = schemaQuerySingleOrMultipleExpand.parse(req.query.expand);
+  const expand = Array.isArray(queryFilterForExpand)
+    ? queryFilterForExpand
+    : queryFilterForExpand
+    ? [queryFilterForExpand]
+    : [];
   const booking = await prisma.booking.findUnique({
     where: { id },
-    include: { attendees: true, user: true, payment: true },
+    include: {
+      attendees: true,
+      user: true,
+      payment: true,
```

Comment:
Not including them as part of the expand feature to avoid breaking changes. expand="team" should serve as a POC.